z3c/dependencychecker/dependencychecker.py

```diff
@@ -25,10 +25,9 @@
 import pkg_resources
 
 from z3c.dependencychecker import importchecker
+from z3c.dependencychecker import logger
 
 
-logger = logging.getLogger(__name__)
```

Comment:
Why this change? It is a very common mechanism.

Reply:
Yes, and I use (and test) it quite a lot! What I did not like is that to get the logger I had to import a module named ``dependencychecker``. When I think about getting the package's logger I either think about that is on ``__init__.py`` or on some utility module, not on a random module that happens to have it.

Does that make sense?

Reply:
You don't need to import some logger, just put the `logger = logging.getLogger(__name__)` line in every file (and a `import logging` import, obviously).

The `.getLogger()` statement doesn't result in an instance you're only allowed to have one from.

Reply:
Yeah, I was doing it just so that I don't have to keep creating it on every file, if you prefer this way, that's fine with me.